screencap.py

```diff
@@ -38,6 +38,14 @@ def qt_warnings():
     environ["QT_SCREEN_SCALE_FACTORS"] = "1"
     environ["QT_SCALE_FACTOR"] = "1"
 
+def showDialog():
```

Comment:
Just one small nitpick - all of the functions are named via snake_case but this one has CamelCase.